lib/sync-srv.js

```diff
@@ -16,6 +16,31 @@ module.exports = function (cfg) {
 
 var syncInited = false;
 
+/**
+ * This is the main export that exposes the fh.sync API.
+ * An instance of the sync API is created when this function is called.
+ * See api.js for where the instance is wired up and exposed as fh.sync.
+ * 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)